src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -11,7 +21,14 @@
 import com.minecolonies.entity.EntityCitizen;
 import com.minecolonies.network.messages.*;
 import com.minecolonies.tileentities.TileEntityColonyBuilding;
-import com.minecolonies.util.*;
+import com.minecolonies.util.BlockPosUtil;
+import com.minecolonies.util.EntityUtils;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.minecolonies.util.EntityUtils'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3AUselessImportCheck)